redis.conf

```diff
@@ -1382,6 +1382,10 @@ disable-thp yes
 # If the AOF is enabled on startup Redis will load the AOF, that is the file
 # with the better durability guarantees.
 #
+# Note that changing this config parameter in an existing database and
+# restarting the server can lead to data loss. A conversion needs to be done
+# interactively through `redis-cli`.
```

Comment:
```suggestion
# Note that changing this value in a config file of an existing database and
# restarting the server can lead to data loss. A conversion needs to be done
# by setting it via CONFIG command on a live server first.
```

Reply:
@oranagra thanks, clear improvement! 👍🏻 I applied the suggested change and squashed the commits.